app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/PieChartViewHolder.java

```diff
@@ -66,13 +68,15 @@ public void setView(ChatMessage model) {
                 pieChart.setRotationAngle(0);
                 pieChart.setDragDecelerationFrictionCoef(0.001f);
                 pieChart.getLegend().setEnabled(false);
-                pieChart.setDescription("");
+                Description description = new Description();
+                description.setText("");
+                pieChart.setDescription(description);
                 RealmList<Datum> datumList = model.getDatumRealmList();
-                final ArrayList<Entry> yVals = new ArrayList<>();
+                final ArrayList<PieEntry> yVals = new ArrayList<>();
```

Comment:
Use List instead of ArrayList

Reply:
@iamareebjamal Should I do this change here in this PR? Because this is an existing file not created by me. All I did was to update the library version and make necessary changes in the constructor to incorporate library revisions.

Reply:
Please create an issue for all such code occurrences

Reply:
Should I consider this under the issue for refactoring code?

Reply:
That would be very big. Let's break refactoring into workable units

Reply:
Make one issue and write todos in parent issue, create those todo issues as you work on them

Reply:
Okay @iamareebjamal . 👍